app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb

```diff
@@ -15,4 +15,13 @@
           contact_detail: contact[:contact_detail],
       }) %>
     <% end %>
+    <% unless object.wrong_contacts.length == 0 %>
```

Comment:
`unless` makes things murky to others when reading 

```erb
<% if object.wrong_contact.present? %>
```

Reply:
I tried using `.present?` and it was not actually filtering. `.length == 0` is filtering properly, but I do agree its ugly if there was a cleaner method.

Reply:
So it is passing an array, so we were able to change this to `.empty?` which gave the same functionality and is much cleaner.